### ISSUE TITLE
Create an EventBus class

### ISSUE THREAD

**Issue by gohabereg** [editor-js/document-model#22]
1. Create an EventBus class
2. Create class or method which would create an event
3. Find a way to pass EventBus to Model nodes (DI?)

**Comment by gohabereg** [comment]
updated
```{
  index: `${DocumentId}:${BlockId}:data@${DataKey}|tune@${TuneName}:${StartIndex}`
  event: ‘added’ | ‘removed’ | ‘modified’,  
  data: 
    text (added, removed) -> new text value
    text (modified) -> end index, new formatting name + value, isInverted
    value (modified) -> new value
    tune (modified) -> new tune value
    block -> BlockSerialized
    document (modified) -> prop name , prop value
}
```

**Comment by gohabereg** [comment]
Closed by #43
